dashboard/test/ui/step_definitions/steps.rb

```diff
@@ -158,6 +179,10 @@ def jquery_is_element_visible(selector)
   wait_with_timeout.until { @browser.execute_script("return $(#{selector.dump}, $(#{parent_selector.dump}).contents()).is(':visible')") }
 end
 
+When /^I wait until Ajax requests are finished$/ do
+  wait_with_short_timeout.until { @browser.execute_script("return $.active == 0") }
```

Comment:
Yeah, this approach will only catch requests using `jquery.ajax()`, which currently includes most of the ajax requests in our system but might be different in the future. I can rename this step to something like "Wait until jQuery ajax requests are finished" to make this more clear